pkg/oc/cli/startbuild/startbuild_test.go

```diff
@@ -27,7 +27,6 @@ import (
 
 	buildv1 "github.com/openshift/api/build/v1"
 	buildclientmanual "github.com/openshift/oc/pkg/helpers/build/client/v1"
-	buildapi "github.com/openshift/origin/pkg/build/apis/build"
 
 	_ "github.com/openshift/origin/pkg/build/apis/build/install"
```

Comment:
This is not addressed, and that's a major problem when you remove it that I'm currently handling.